src/Redux/APIFunctions/reservations.js

```diff
@@ -0,0 +1,55 @@
+// Fetch data from the backend API
+import { ActionTypes } from '../Actions/reservations';
+
+const URL = 'https://stark-badlands-38572.herokuapp.com/api/v1/reservations';
+
+export const fetchReservations = (reservation) => ({
+  type: ActionTypes.FETCH_RESERVATIONS,
+  payload: reservation,
+});
+
+export const fetchReservationsLoading = () => ({
+  type: ActionTypes.FETCH_DATA_LOADING,
+});
+
+export const fetchReservationsError = () => ({
+  type: ActionTypes.FETCH_DATA_ERROR,
+});
+
+export const fetchAddedReservation = (reservation) => ({
+  type: ActionTypes.ADD_RESERVATION,
+  payload: reservation,
+});
+
+export const getReservations = () => (dispatch) => {
+  dispatch(fetchReservationsLoading());
+  fetch(URL)
+    .then((response) => response.json())
+    .then((data) => {
+      dispatch(fetchReservations(data));
+    })
+    .catch(() => {
+      dispatch(fetchReservationsError());
+    });
+};
+
+const reservationURL = 'https://stark-badlands-38572.herokuapp.com/api/v1/hotels';
+
+export const postReservation = (reservation, hotelId) => (dispatch) => {
+  console.log(reservation);
```

Comment:
- [ ] According to the general [Javascript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) best practices requirements, please kindly don't push to the repo any debugging code `console.log`.  

Your project repo has many of them (too many to list here).  

Please, kindly use the search function on your code editor, and remove them all, in order to improve the cleanliness of the repo 💪